src/script/lua_api/l_object.cpp

```diff
@@ -738,6 +738,28 @@ int ObjectRef::l_get_attach(lua_State *L)
 	return 5;
 }
 
+// get_children(self)
+int ObjectRef::l_get_children(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	ObjectRef *ref = checkobject(L, 1);
+	ServerActiveObject *sao = getobject(ref);
+	if (sao == NULL)
+		return 0;
+
+	//Do it
```

Comment:
I suggest deleting this line

Reply:
The `//Do it`? I saw it in every other function so I thought I had to replicate it. As I'm planning to clean this class up a bit, should I remove the do it everywhere?

Reply:
Yes, preferably in a separate refactor PR.